### ISSUE TITLE
have all steps expanded?

### ISSUE THREAD

**Issue by rgarcia** [damianricobelli/shadcn-stepper#2]
Hi there--love this component and getting some good use out of it. I'd love for there to be a way with the vertical stepper to show all the steps as expanded. This is useful in cases where you'd like the user to see all the steps at once, without hiding any of the content / form entries contained within a step.

**Comment by damianricobelli** [comment]
Hey, thanks for your comment @rgarcia 

In order to do that, you can send the `expandVerticalSteps` prop to `true` inside the Stepper component.

```tsx
<Stepper expandVerticalSteps>
  ...
</Stepper>
```

**Comment by rgarcia** [comment]
amazing--don't know how I missed that. Thanks a lot!

**Comment by dgdavid** [comment]
Hi there!

As far as I can see, such a prop is no longer available after changes made in commit c91c21733bf6e3b21064ce0fad9d1a4b7df7 (Aug 15, 2024), where _components/stepper/context.tsx‎_ was removed

```diff
- interface StepperContextValue extends StepperProps {
-   clickable?: boolean
-   isError?: boolean
-   isLoading?: boolean
-   isVertical?: boolean
-   stepCount?: number
-   expandVerticalSteps?: boolean
-   activeStep: number
-   initialStep: number
- }
```

Is possible to keep expanding all vertical steps?
